oauth2_http/javatests/com/google/auth/oauth2/DownscopedCredentialsTest.java

```diff
@@ -107,11 +109,35 @@ public void refreshAccessToken() throws IOException {
         "urn:ietf:params:oauth:token-type:access_token", query.get("requested_token_type"));
   }
 
+  @Test
+  public void refreshAccessToken_userCredentials_expectExpiresInCopied() throws IOException {
```

Comment:
add doc explaining the behavior that you are trying to test... from PR description :)